admin/package-files/docker-plugin/systemd/flocker-docker-plugin.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Flocker Docker Plugin
+Before=docker.service
+
+[Service]
+ExecStart=/usr/sbin/flocker-docker-plugin
+Restart=always
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
Actually, no, I didn't put that in because it does not in fact require Docker. It can run fine without it. Theoretically someone else could implement the Docker Volume plugin API client and talk to our plugin.